### PR TITLE
Replaced "validation set" with "test set" in step 8

### LINKED ISSUE #202: Clarify train test and validation data
https://github.com/carpentries-incubator/deep-learning-intro/blob/2959e87389a374b48d376c66bfe9b874ad41d51f/_episodes/01-introduction.md?plain=1#L156

> Sometimes we'll also use a 3rd set known as a validation set to check our results after training is complete

validation data is used _during_ training, e.g. in ep03

https://github.com/carpentries-incubator/deep-learning-intro/blob/2959e87389a374b48d376c66bfe9b874ad41d51f/_episodes/03-monitor-the-model.md?plain=1#L540-L543

### PR DESCRIPTION
**Pull request by dsmits**
Closes #202

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -184,11 +184,11 @@ of the network.
 
 ### 8. Measure Performance
 
-Once we think the network is performing well we want to measure its performance. To do this we can use some additional data that wasn't part of the training, this is known as a validation set. There are many different methods available for doing this and which one is best depends on the type of task we are attempting. These metrics are often published as an indication of how well our network performs.
+Once we think the network is performing well we want to measure its performance. To do this we use some additional data that wasn't part of the training, this is known as a test set. There are many different methods available for measuring performance and which one is best depends on the type of task we are attempting. These metrics are often published as an indication of how well our network performs.
```
Comment (colinsauze) [inline _episodes/01-introduction.md:187]: This kind of implies that the test set isn't being used during the training process. We should at at least remove the first sentence to make this clearer.

Comment (dsmits) [reply _episodes/01-introduction.md:187]: Yeah I get what you mean. How about replacing it with "Once we have trained the network we want to measure it's performance." ? What do you think @colinsauze ?

Comment (psteinb) [reply _episodes/01-introduction.md:187]: I agree with @colinsauze. To me, the __test set__ is essential during training. The current formulation reads as if the test set is used after the `fit` method returns.

```suggestion
Assessing the quality of predictions of our model is essential during training and after. To do this during training, we use some additional data that wasn't part of the training in order to obtain an objective assessment. This is known as a test set. 

There are many different metrics available for measuring performance. Which one is best depends on the type of task we are attempting. These metrics are often published as an indication of how well our network performs.
```

Comment (dsmits) [reply _episodes/01-introduction.md:187]: The terminology is not very consistent in the field. Sometimes what you're referring to is the test set, sometimes it is the other way around.

However, in the [`fit` function of keras](https://keras.io/api/models/model_training_apis/#fit-method) there is a parameter called `validation_data` to monitor performance during training. I propose we stick to the keras terminology.

Comment (psteinb) [reply _episodes/01-introduction.md:187]: Totally in favor with @dsmits.

Comment (colinsauze) [reply _episodes/01-introduction.md:187]: Are we going to add a callout box explaining how these terms are often interchanged? Is that best here in the introduction or in episode 3 when we use it properly?

Comment (dsmits) [reply _episodes/01-introduction.md]: @colinsauze I have added it to episode 3 where the term is introduced properly.

Comment (dsmits) [reply _episodes/01-introduction.md]: Let me know if you approve

File: _episodes/03-monitor-the-model.md [hunk 2]
```diff
@@ -431,6 +431,14 @@ As we saw when comparing the predictions for the training and the test set, deep
 This is the *validation set* which can be regarded as a second test set. As with the test set, the datapoints of the *validation set* are not used for the actual model training itself. Instead, we evaluate the model with the *validation set* after every epoch during training, for instance to stop if we see signs of clear overfitting.
 Since we are adapting our model (tuning our hyperparameters) based on this validation set, it is *very* important that it is kept separate from the test set. If we used the same set, we wouldn't know whether our model truly generalizes or is only overfitting.   
 
+> ## Training vs. validation set
+> Not everybody agrees on the terminology of training set versus validation set. You might find 
+> examples in literature where these terms are used the other way around.
+> 
+> We are sticking to the definition that is consistent with the Keras api. In there, the validation
```
Comment (colinsauze) [inline _episodes/03-monitor-the-model.md:438]: API should be in upper case

Comment (psteinb) [reply _episodes/03-monitor-the-model.md:438]:
```suggestion
> We are sticking to the definition that is consistent with the Keras API. In there, the validation 
```
